tox.ini

```diff
@@ -95,3 +95,29 @@ setenv =
 commands =
     pip install -e .
 	behave --tags=-disabled integration_tests
+
+
+[testenv:ci_gsye_122]
+basepython = python3.8
+passenv = LANG TERM LANGUAGE LC_ALL LD_LIBRARY_PATH BRANCH
+deps = pip-tools
+whitelist_externals = git
+                      /bin/rm
+                      /bin/cp
+                      /bin/mkdir
+                      echo
+					  bash
+commands =
+		python -c "import fcntl; fcntl.fcntl(1, fcntl.F_SETFL, 0)"
+		pip-sync requirements/tests.txt requirements/pandapower.txt
+		pip install -e .
+		pip uninstall -y gsy-framework
+		pip install git+https://github.com/gridsingularity/gsy-framework@{env:BRANCH:master}
+		rm -rf gsy-backend-integration-tests
+		rm -rf integration_tests
```

Comment:
You do not need these as the worker will not have any artefacts from old runs in it.

Reply:
Oh, I see. However I recall running the ci command locally. Will this not be a possibility anymore??